src/metatensor/models/utils/per_atom.py

```diff
@@ -1,5 +1,43 @@
+from typing import Dict, List
+
 import torch
 from metatensor.torch import TensorBlock, TensorMap
+from metatensor.torch.atomistic import System
+
+
+def average_predictions_and_targets_by_num_atoms(
+    predictions: Dict[str, TensorMap],
+    targets: Dict[str, TensorMap],
```

Comment:
Why does this function need both. `predictions` and `targets`. Wouldn't it be clearer to have a function that takes only one `tensormap_dict_list: Dict[str, TensorMap]` and we call this function twice in the code

Reply:
You're right. My bad